docs/how_to_create_a_plugin.md

```diff
@@ -1,228 +1,161 @@
+        # Add your global settings
+        # Available types: :integer, :boolean
+        # settings.attribute :vote_limit, type: :integer, default: 0
+      end
 
-```ruby
-# frozen_string_literal: true
-require "rails"
-require "active_support/all"
+      feature.settings(:step) do |settings|
+        # Add your settings per step
+      end
 
-require "decidim/core"
-require "jquery-rails"
-require "sass-rails"
-require "foundation-rails"
-require "foundation_rails_helper"
-require "autoprefixer-rails"
+      feature.register_resource do |resource|
+        # Register a optional resource that can be references from other resources.
+        # resource.model_class_name = "Decidim::<EngineName>::<ResourceName>"
+        # resource.template = "decidim/<engine_name>/<resource_view_folder>/linked_<resource_name_plural>"
+      end
 
-module Decidim
-  module <EngineName>
-    # Decidim's core Rails Engine.
-    class Engine < ::Rails::Engine
-      isolate_namespace Decidim::<EngineName>
+      feature.register_stat :some_stat do |features, start_at, end_at|
+        # Register some stat number to the application
+      end
 
-      initializer "decidim_<engine_name>.assets" do |app|
-        app.config.assets.precompile += %w(decidim_<engine_name>_manifest.js)
+      feature.seeds do
+        # Add some seeds for this feature
       end
     end
-  end
-end
-```
-
-9. Add `lib/decidim/<engine_name>/feature.rb` with this:
-
-```ruby
-# frozen_string_literal: true
-
-require_dependency "decidim/features/namer"
-
-Decidim.register_feature(:<engine_name>) do |feature|
-  feature.engine = Decidim::<EngineName>::Engine
-  feature.admin_engine = Decidim::<EngineName>::AdminEngine
-  feature.icon = "decidim/<engine_name>/icon.svg"
-
-  feature.on(:before_destroy) do |instance|
-    # Code executed before removing the feature
-  end
-
-  # These actions permissions can be configured in the admin panel
-  feature.actions = %w()
-
-  feature.settings(:global) do |settings|
-    # Add your global settings
-    # Available types: :integer, :boolean
-    # settings.attribute :vote_limit, type: :integer, default: 0
-  end
-
-  feature.settings(:step) do |settings|
-    # Add your settings per step
-  end
-
-  feature.register_resource do |resource|
-    # Register a optional resource that can be references from other resources.
-    # resource.model_class_name = "Decidim::<EngineName>::<ResourceName>"
-    # resource.template = "decidim/<engine_name>/<resource_view_folder>/linked_<resource_name_plural>"
-  end
+    ```
 
-  feature.register_stat :some_stat do |features, start_at, end_at|
-    # Register some stat number to the application
-  end
+1. Replace `Rakefile` with:
 
-  feature.seeds do
-    # Add some seeds for this feature
-  end
-end
-```
+    ```ruby
+    # frozen_string_literal: true
+    require "decidim/common_rake"
+    ```
 
-10. Replace `Rakefile` with:
+1. Remove `MIT-LICENSE` and change `README`
```

Comment:
Yup. `rails plugin new` automatically generates this `MIT-LICENSE` file, so we just remove it. This works because the plugins generated by these instructions are meant to live inside the repo, so the global AGPL3 is assumed. We should tend to decouple this though, so the instructions are good for "external plugins" too.